XCUITests/HistoryTests.swift

```diff
@@ -227,4 +231,18 @@ class HistoryTests: BaseTestCase {
         XCTAssertFalse(app.cells.staticTexts["Recently Closed"].isSelected)
         waitForNoExistence(app.tables["Recently Closed Tabs List"])
     }
+
+    func testClearRecentHistory() {
+        navigator.performAction(Action.ClearLastHourHistory)
```

Comment:
Please add some comments like // data will not be removed since it is older than today or yesterday...